sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/SdkMeterProviderTest.java

```diff
@@ -748,22 +748,21 @@ void viewSdk_capturesBaggageFromContext() {
   }
 
   @Test
-  void sdkMeterProvider_supportsMultipleCollectorsCumulative() {
```

Comment:
These two tests were written in a way that allowed the bug to hide. Recording additional measurements with attributes reveals the true behavior. These tests as currently written fail on `main`.